vslib/src/SwitchStateBase.cpp

```diff
@@ -1851,6 +1851,20 @@ sai_status_t SwitchStateBase::refresh_macsec_sci_in_ingress_macsec_acl(
     return SAI_STATUS_SUCCESS;
 }
 
+sai_status_t SwitchStateBase::refresh_queue_pause_status(
+        _In_ sai_object_id_t object_id)
+{
+    SWSS_LOG_ENTER();
+
+    sai_attribute_t attr;
+    attr.id = SAI_QUEUE_ATTR_PAUSE_STATUS;
+    attr.value.booldata = false;
```

Comment:
is it always false? should that status depend on some actual queue internal status ?

Reply:
The attribute needs to be present in COUNTERS_DB, but the actual value is dummy to DEBUG_STORM mode.

Reply:
could you put that comment there in code ? that this is dummy?

Reply:
Done